web/src/lib/models/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './player.model';
+export * from './bot.model';
+export * from './match_result.model';
```

Comment:
This allows us to do this: `import { Player, Bot } from "$lib/models";`, instead of ending up with this:

```typescript
import Player from "$lib/models/player";
import Bot from "$lib/models/bot";
// ...
```

which can end up being really long and not really nice to change if needed. The re-exports in `index.ts` makes this a lot nicer: you don't have to remember which file contains what.

The `index.ts` (or `index.js`) is treated somewhat specially in Node.js (not JavaScript in general, just Node.js): it is treated as the root file of a directory, sort of like what `__init__.py` is in Python (Python took inspiration from this iirc).